inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -808,16 +808,35 @@ def affiliations_for_hidden_collections(obj):
     return [affiliations_mapping[affiliation] for affiliation in affiliations_set]
 
 
+def reports_for_hidden_collections(obj):
+    hidden_collection_mapping = current_app.config.get(
+        "AFFILIATIONS_TO_HIDDEN_COLLECTIONS_MAPPING", {}
+    )
+    hidden_collections = set()
+    for report_number in obj.data.get("report_number", []):
+        hidden_collections.update(
+            [
+                match.upper()
```

Comment:
if I'm not mistaken, if there is no match it will fail, `None` doesn't have `upper`

Reply:
No, I am using `re.findall` there which is returning empty list if there's no match